api/idea-card/models/idea-card.settings.json

```diff
@@ -0,0 +1,59 @@
+{
+  "kind": "collectionType",
+  "collectionName": "idea_cards",
+  "info": {
+    "name": "IdeaCard"
+  },
+  "options": {
+    "increments": true,
+    "timestamps": true,
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "ideaName": {
+      "type": "string",
+      "required": true,
+      "unique": true
+    },
+    "tagline": {
+      "type": "text",
+      "required": true
+    },
+    "description": {
+      "type": "text",
+      "required": true
+    },
+    "targetAudience": {
+      "type": "text",
+      "required": true
+    },
+    "timeCommitmentMin": {
+      "type": "integer",
+      "required": true
+    },
+    "timeCommitmentMax": {
+      "type": "integer",
+      "required": true
+    },
+    "difficultyLevel": {
```

Comment:
What are the possible values? I think this is better modeled as an enum.

Reply:
Hania said beginner, intermediate, and advanced. I could see this becoming an enum. Let me ask her if those are the only values that field will have.

Reply:
Ok she said only those 3. I’ll change this to being an enum.